tools/mo/openvino/tools/mo/utils/ir_engine/ir_engine.py

```diff
@@ -5,14 +5,14 @@
 import logging as log
 import os
 import sys
-
-from defusedxml import defuse_stdlib
-import defusedxml.ElementTree as ET
 from argparse import Namespace
+from collections import OrderedDict
```

Comment:
could you just add OrderedDict in next line?

Reply:
Yes, done.